client/index.html

```diff
@@ -3,6 +3,11 @@
   <meta charset="UTF-8">
   <title>SuttaCentral</title>
   <link rel="icon" href="/img/favicon.ico">
+  <link rel="apple-touch-icon" sizes="180x180" href="/img/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="/img/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="/img/favicon-16x16.png">
+  <link rel="manifest" href="/img/manifest.json">
```

Comment:
The manifest should not reside in the `img` directory, please move it up. When you do that, you'll have to update the paths in the manifest as well.